packages/pds/service/index.js

```diff
@@ -78,9 +78,9 @@ const main = async () => {
   })
 }
 
-const pgUrl = ({ username, password, host, port }) => {
+const pgUrl = ({ username = "postgres", password = "postgres", host = "0.0.0.0", port = "5432", database = "postgres", sslmode }) => {
```

Comment:
Is there a reason we should be using `0.0.0.0` rather than `localhost` or `127.0.0.1`?  I guess I usually think of `0.0.0.0` more as a source address than a destination.

Reply:
no particular reason. localhost or 127.0.0.1 is probably the better default

Reply:
Cool!
```suggestion
const pgUrl = ({ username = "postgres", password = "postgres", host = "localhost", port = "5432", database = "postgres", sslmode }) => {
```

Reply:
It might not be relevant in this context but worth noting -- there's a meaningful semantic difference between 0.0.0.0 and 127.0.0.1. 

0.0.0.0 will bind to some definition of all interfaces - e.g. eth0, lo0.

127.0.0.1 will only bind to lo0/loopback.

i.e. it's possible to accidentally exposure services intended to only be accessible from loopback, if 0.0.0.0 is used where 127.0.0.1 should be.

It looks like localhost is just a DNS name for 127.0.0.1.

[sample stackoverflow answer](https://stackoverflow.com/a/20778887/1234621) w/ more info.

Reply:
whoa, good to know! thanks for taking the time to explain all of this